main.tf

```diff
@@ -1043,6 +1132,59 @@ resource "aws_network_acl_rule" "elasticache_outbound" {
   ipv6_cidr_block = lookup(var.elasticache_outbound_acl_rules[count.index], "ipv6_cidr_block", null)
 }
 
+################################################################################
+# Firewall Network ACLs
+################################################################################
+resource "aws_network_acl" "firewall" {
+  count = var.create_vpc && var.firewall_dedicated_network_acl && length(var.firewall_subnets) > 0 ? 1 : 0
+
+  vpc_id     = element(concat(aws_vpc.this.*.id, [""]), 0)
+  subnet_ids = aws_subnet.firewall.*.id
+
+  tags = merge(
+    {
+      "Name" = format("%s-${var.firewall_subnet_suffix}", var.name)
+    },
+    var.tags,
+    var.firewall_acl_tags,
+  )
+
+  lifecycle {
+    ignore_changes = [
+      tags["Owner"],
+      tags["Type"],
+    ]
+  }
```

Comment:
Changes to network ACL doesn't require replacement (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-network-acl.html). Why are we ignoring these?

Reply:
ah my bad, will remove these